node/executor/src/lib.rs

```diff
@@ -80,11 +80,10 @@ mod tests {
 
 	type TestExternalities<H> = CoreTestExternalities<H, u64>;
 
-	// TODO: fix for being charged based on weight now.
 	fn transfer_fee<E: Encode>(extrinsic: &E) -> Balance {
 		<TransactionBaseFee as Get<Balance>>::get() +
-			<TransactionByteFee as Get<Balance>>::get() *
-			(extrinsic.encode().len() as Balance)
+		<TransactionByteFee as Get<Balance>>::get() *
```

Comment:
you reverted my whitespace fix. these two lines are run-on and need to be indented.